mne_lsl/stream/stream_lsl.py

```diff
@@ -250,6 +256,18 @@ def _acquire(self) -> None:
                 data_ref = data[:, self._ref_channels].mean(axis=1, keepdims=True)
                 data[:, self._ref_from] -= data_ref
 
+            # apply filters on (n_times, n_channels) data
+            for filter_ in self._filters:
+                if filter_["zi"] is None:
+                    # initial conditions are set to a step response steady-state set
+                    # on the mean on the acquisition window (e.g. DC offset for EEGs)
+                    filter_["zi"] = filter_["zi_coeff"] * np.mean(
+                        data[filter_["picks"]], axis=0
+                    )
+                data, filter_["zi"] = sosfilt(
+                    filter_["sos"], data[filter_["picks"]], zi=filter_["zi"], axis=0
+                )
```

Comment:
If the inital conditions have not been initialized, multiply the `sosfilt_zi(sos)[..., np.newaxis]` with the mean of the signal to model the step response.

What about the case of multiple filters applied to the same channel? 
First iteration:
- filter 1 applied to channel 1 -> zi1
- filter 2 applied to channel 2 -> zi2 -> is zi1 still valid?